### PR TITLE
Read max exif header from s3

### PR DESCRIPTION
**Pull request by elliott-imhoff**
# Read max exif header from s3
## What?
Previously I was trying to minimize the amount of data we had to read from s3 by first reading the first 12 bytes of the image to parse the size of the exif header, then reading the exact amount of bytes to get the entire header. However it seems this wasn't quite enough data, as we were getting errors for some images when parsing their exif from s3. It seems adding an extra 4 bytes was enough to fix it, but I did some speed tests and reading the maximum size of the exif header (65536 bytes) seems to be very close to the same speed as making two different, smaller reads. So I decided to just hard code it to the maximum size and leave it at that.
## Why?
Fixes an error with auto-rejection in the mfstand tool
## PR Checklist
- [x] Merged latest master
- [x] Updated version number
- [x] Version numbers match between package ``_version.py`` and *pyproject.toml*

### CODE REVIEW

File: imgparse/getters.py [hunk 1]
```diff
@@ -65,18 +65,9 @@ def read_exif_header_from_s3(image_path):
...
-    # See here for more info: https://www.media.mit.edu/pia/Research/deepview/exif.html
-    if exif_start[:8] == "ffd8ffe1" and exif_start[12:] == "457869660000":
-        size = int(exif_start[8:12], 16)
-    else:
-        logger.warning("S3 file doesn't match expected exif header format")
-        size = 65536
 
     # Read the entire exif header for the image and return it to be parsed by exifread
-    return BytesIO(obj.get(Range=f"bytes=0-{size}")["Body"].read())
+    return BytesIO(obj.get(Range="bytes=0-65536")["Body"].read())
```
Comment (dzermas) [inline imgparse/getters.py:70]: Double checked with Zach and it seems that 64kb should be more than enough for all of our images including 6X.
